ci/makefiles/osx.mk

```diff
@@ -0,0 +1,23 @@
+# installs java 1.8, android's SDK/NDK, cython and p4a
+
+# The following variable/s can be override when running the file
+ANDROID_HOME ?= $(HOME)/.android
+
+all: install_java upgrade_cython install_android_ndk_sdk install_p4a
+
+install_java:
```

Comment:
This is to think about later, but installing system requirements could actually be part Linux Makefile as well